docs/history.rst

```diff
@@ -10,6 +10,7 @@ Latest
 0.3.2
 -----
 - BUG: handle lock=True in open_rasterio (issue #273)
+- ENH: enable `engine="rasterio"` via xarray backend API (issue #197 pull #281)
```

Comment:
If you rebase with the changes in master from #204, you can add this to the `Lastest` section.